File: tourneyAPI/Models/Entities/Characters/Terry.cs
namespace Entities;

using Enums;
using Enums;

// Defines the competitive profile metadata for this playable character.
public class Terry : Character
{
    // Initializes this character's default competitive attributes for matchmaking and tier logic.
    public Terry()
    {
        Id = CharacterId.Terry;
        characterName = CharacterName.TERRY;
        archetype = Archetype.FOOTSIES;
        fallSpeed = FallSpeed.FAST_FALLERS;
        weightClass = WeightClass.MIDDLEWEIGHT;
        tierPlacement = TierPlacement.A;
    }

}

File: tourneyClient/src/models/entities/Characters/Terry.ts
import { CharacterId } from "@/models/Enums/CharacterId";
import { Character } from "../Character"
import { CharacterName } from "@/models/Enums/CharacterName";
import { Archetype } from "@/models/Enums/Archetype";
import { FallSpeed } from "@/models/Enums/FallSpeed";
import { WeightClass } from "@/models/Enums/WeightClass";
import { TierPlacement } from "@/models/Enums/TierPlacement";

export const Terry: Character =
{
    id: CharacterId.Terry,
    characterName: CharacterName.TERRY,
    archetype: Archetype.FOOTSIES,
    fallSpeed: FallSpeed.FAST_FALLERS,
    weightClass: WeightClass.MIDDLEWEIGHT,
    tierPlacement: TierPlacement.A
}

export default Terry; 